cmd2/cmd2.py

```diff
@@ -22,30 +22,18 @@
 
 Git repository on GitHub at https://github.com/python-cmd2/cmd2
 """
+# many imports are lazy-loaded when they are needed
 import argparse
-import atexit
 import cmd
-import codecs
 import collections
 from colorama import Fore
-import copy
-import datetime
-import functools
 import glob
-import io
-from io import StringIO
 import os
 import platform
 import re
 import shlex
-import signal
-import subprocess
 import sys
-import tempfile
-import traceback
 from typing import Callable, List, Optional, Union, Tuple
```

Comment:
I think ``Optional`` is now an unused import

Reply:
Yup.